Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ruby:2.7-slim
+
+LABEL Name=morandirb Version=0.0.1
```

Comment:
This is only the Docker image version, I guess?

Reply:
yeah, the image. this was in the dockerfile when mark set everything up. I don't know if this is typical for livelink dockerfiles or if you'd prefer something else?